CONTRIBUTING.md

```diff
@@ -22,24 +22,46 @@ See the following directory trees for further recommended examples:
 - [test/built-ins/WeakMap](./test/built-ins/WeakMap/)
 - [test/language/arguments-object](./test/language/arguments-object/)
 
-**Note** The project is currently transitioning from a naming system based on specification section numbers. There remains a substantial number of tests that conform to this outdated convention; contributors should ignore that approach when introducing new tests and instead encode this information using [the `esid` frontmatter tag](#esid).
+**Note** The project is currently transitioning from a naming system based on specification section numbers. There remains a substantial number of tests that conform to this outdated convention; contributors should ignore that approach when introducing new tests and instead encode this information using [the `esid` frontmatter key](#esid).
 
 ## Test Case Style
 
 A test file has three sections: Copyright, Frontmatter, and Body.  A test looks roughly like this:
 
 ```javascript
-// Copyright (C) 2015 [Contributor Name]. All rights reserved.
+// Copyright (C) $Year $ContributorName. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
-
 /*---
- description: brief description
- info: >
-   verbose test description, multiple lines OK.
-   (info typically contains relevant, direct quote from ECMAScript)
+description: >
+    brief description, e.g. "Non-numeric input must be rejected with a TypeError"
+esid: reference to spec section, e.g. "sec-well-known-symbols"
+info: |
+    verbose test description, multiple lines OK.
+    (info should contain relevant, direct quotes from ECMAScript if possible)
+    For example:
+
+    String.fromCodePoint ( ..._codePoints_ )
+
+    5. Repeat, while _nextIndex_ &lt; _length_
+      a. Let _next_ be _codePoints_[_nextIndex_].
+      b. Let _nextCP_ be ? ToNumber(_next_).
+      d. If _nextCP_ &lt; 0 or _nextCP_ &gt; 0x10FFFF, throw a *RangeError* exception.
+features: [example]
 ---*/
 
-[Test Code]
+var maxCodePoint = 0x10FFFF;
+var maxCodePointString = '\u{10FFFF}';
+
+assert.sameValue(String.fromCodePoint(maxCodePoint), maxCodePointString,
+    'String.fromCodePoint can produce a string including the highest code point');
+
+assert.throws(
+  RangeError,
+  function() {
+    String.fromCodePoint(maxCodePoint + 1);
```

Comment:
Am I correct in ascertaining that the preferred string quote character is `'` and the preferred indentation is two spaces?

Reply:
Preferred is subjective.

I'm all up for 2 spaces, no hard tabs.

The single quote character is more often used by contributors to the project, so I'd be fine to observe this convention.